src/components/_General/buttons.tsx

```diff
@@ -23,6 +23,18 @@ const getTheme = theme => {
       return `
         background: var(--color-almostBlack);
         border: 1px solid var(--color-lighterBlack);`;
+    case Colors.DANGER:
```

Comment:
Can we avoid `switch` statements? They make it easy to break stuff and are usually less readable than the alternatives ([ref](https://medium.com/tandemly/whats-wrong-with-the-switch-statement-in-javascript-c560e8ea3c0b), [ref](https://ericleads.wordpress.com/2012/12/11/switch-case-considered-harmful/)).

Reply:
this was here before though. I'll refactor to if/elses

Reply:
joking. I'll turn that into an object.

Reply:
whew you had me there.